server/schemas/integration.graphql

```diff
@@ -0,0 +1,56 @@
+enum IntegrationType {
+  Public
+  Private
+}
+
+type Integration implements Node {
+  id: ID!
+  name: String!
+  description: String
+  logoUrl: URL!
+  type: IntegrationType!
+  token: String!
+  developerId: ID!
+  developer: User!
+  webhook: [Webhook!]!
+  createdAt: DateTime!
+  updatedAt: DateTime!
+}
```

Comment:
```suggestion
type Integration implements Node {
  id: ID!
  name: String!
  description: String
  logoUrl: URL!
  type: IntegrationType!
  developerId: ID!
  developer: User!
  createdAt: DateTime!
  updatedAt: DateTime!
  config: IntegrationConfig # only for a developer of this integration
}

type IntegrationConfig {
  token: String!
  webhooks: [Webhook!]!
}
```

- I have divided the information into different types that should only be visible to the developer himself.
- "type" is not usable as a field name?